server/src/test/java/org/solarbank/server/unit/CalculateServiceTest.java

```diff
@@ -0,0 +1,35 @@
+package org.solarbank.server.unit;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.solarbank.server.dto.UserInputDto;
+import org.solarbank.server.service.CalculateService;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.Map;
+
+public class CalculateServiceTest {
+
+    private CalculateService calculateService;
+
+    @BeforeEach
+    public void setUp() {
+        calculateService = new CalculateService();
+    }
+
+    @Test
+    public void testProcessUserInput() {
```

Comment:
There are many naming strategies for tests but my personal fav is given, when, then.

For example in this test you could call it something like:

`calculateRequest_validInputs_returnsExpectedResult`

Reply:
changed test name